### PR TITLE
beta change log update

### PR DESCRIPTION
**Pull request by ccfishk**
**What this PR does / why we need it**:
  log build changes accordingly.
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
  documentation
**Special notes for your reviewer**:

**PR Readiness Checklist**:

Complete these before marking the PR as `ready to review`:

- [ ] the `CHANGELOG.md` release notes have been updated to reflect any significant (and particularly user-facing) changes introduced by this PR

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,6 +36,10 @@
 
  ## [2.0.0] - TBA
 
+#### Hot-Fix
+- Fixed controller metrics expose bug
+  [#1497] (https://github.com/Kong/kubernetes-ingress-controller/issues/1497)
```
Comment (mflendrich) [inline CHANGELOG.md:41]:
```suggestion
#### Fixed
- Prometheus metrics were not exposed on the metrics endpoint in 2.0.0-beta.1 by default [#1497](https://github.com/Kong/kubernetes-ingress-controller/issues/1497)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # Table of Contents
 -  [2.0.0](#200---tbd)
+ - [2.0.0-beta.2](#200-beta2---202108015)
```
Comment (mflendrich) [inline CHANGELOG.md:3]: This is only a table of contents - each entry here needs to correspond to a section later in the document.

There is no `2.0.0-beta.2` section in the document yet.

File: CHANGELOG.md [hunk 3]
```diff
@@ -36,6 +37,11 @@
 
  ## [2.0.0] - TBA
 
+## [2.0.0-beta.2] - 2021/08/15
```
Comment (mflendrich) [inline CHANGELOG.md:40]: 15 or 16?

File: CHANGELOG.md (hunk 2 above)
Comment (mflendrich) [reply CHANGELOG.md:3]: broken link:
```suggestion
 - [2.0.0-beta.2](#200-beta2---20210815)
```

also be mindful of the comment below - the release date for beta.2 will probably be the 16th, not the 15th

File: CHANGELOG.md [hunk 4]
```diff
@@ -1276,6 +1282,7 @@ Please read the changelog and test in your environment.
  - The initial versions  were rapildy iterated to deliver
    a working ingress controller.
 
+[2.0.0-beta.2]: https://github.com/kong/kubernetes-ingress-controller/compare/2.0.0-alpha.3...2.0.0-beta.2
```
Comment (mflendrich) [inline CHANGELOG.md:1285]:
```suggestion
[2.0.0-beta.2]: https://github.com/kong/kubernetes-ingress-controller/compare/2.0.0-beta.1...2.0.0-beta.2
```

File: CHANGELOG.md (hunk 3 above)
Comment (ccfishk) [reply CHANGELOG.md]: 40d3323053438e2463d43013229052622eedde34

**Review by mflendrich — APPROVED** [review]

**Pull request merged**